.gitlab/source_test/macos.yml

```diff
@@ -2,39 +2,6 @@
 include:
   - .gitlab/common/macos.yml
 
-tests_macos:
-  stage: source_test
-  # HACK: Run macOS unit tests only on full pipelines, to limit the use of macOS GitHub runners.
-  rules:
-    - !reference [.on_main_or_release_branch_or_all_builds_or_pkg_installer_changes]
```

Comment:
I _think_ this is the only place where this rule was used. Could you check and clean up?

Reply:
Good point, fixed with [6c93173](https://github.com/DataDog/datadog-agent/pull/32151/commits/6c931736c37abec27954eb22912e7aab8075120f) !